src/Payments/PaymentRequestButtonElement.res

```diff
@@ -39,6 +39,7 @@ module WalletsSaveDetailsText = {
 @react.component
 let make = (~sessions, ~walletOptions, ~paymentType) => {
   open SessionsType
+  let methodslist = Recoil.useRecoilValueFromAtom(RecoilAtoms.paymentMethodList)
```

Comment:
You can use `paymentMethodListValue` instead of `paymentMethodList`

Reply:
updated